external/CMakeLists.txt

```diff
@@ -171,20 +177,6 @@ target_compile_options(Jolt PRIVATE
         -Wno-error  # Do not treat warnings as errors
 )
 
-# GoogleTest
```

Comment:
No more google test?

Reply:
Oh I am declaring Google test in the UnitTesting project instead of in the External one, should remove that comment